### PR TITLE
Add stylelint-formatter-gitlab-code-quality-report

### PR DESCRIPTION
**Pull request by tijsverkoyen**
> Which issue, if any, is this issue related to?

None

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,7 @@
 - [stylelint-checkstyle-formatter](https://www.npmjs.com/package/stylelint-checkstyle-formatter) - Output Checkstyle XML reports.
 - [stylelint-codeframe-formatter](https://www.npmjs.com/package/stylelint-codeframe-formatter) - Output codeframe errors and warnings.
 - [stylelint-formatter-github](https://www.npmjs.com/package/stylelint-formatter-github) - Output errors and warnings directly in GitHub pull requests.
+- [stylelint-formatter-gitlab-code-quality-report](https://www.npmjs.com/package/stylelint-formatter-gitlab-code-quality-report) - Output errors and warnings as a Gitlab Code Quality report (artifact).
```
Comment (ybiquitous) [inline README.md:74]: [suggest] Maybe a typo?

```suggestion
- [stylelint-formatter-gitlab-code-quality-report](https://www.npmjs.com/package/stylelint-formatter-gitlab-code-quality-report) - Output errors and warnings as a GitLab Code Quality report (artifact).
```

**Review by ybiquitous — COMMENTED** [review]
@tijsverkoyen Thank you. Almost LGTM. Can you confirm my review comment?

File: README.md (hunk 1 above)
Comment (tijsverkoyen) [reply README.md]: Fixed in 2503e77b1a1a5e18b315e60fafc2e128b7a69066

**Review by ybiquitous — APPROVED** [review]
Thanks!

**Pull request merged**